client/app/services/services.js

```diff
@@ -309,20 +309,20 @@ angular.module('spkr.services', [])
     function draw(){
```

Comment:
This line comment is actually a file comment, and not for a change that was made, but for a change that hasn't been made yet.

Could we modify this comment to explain the current shortcomings of the authentication, and remove the 'don't touch' at the beginning of this document to 'please touch', basically? 'Don't touch _x_' looks bad in code, especially when it's a label above poorly functioning code.